OmniBLE/PumpManagerUI/ViewControllers/OmnipodSettingsViewController.swift

```diff
@@ -317,188 +316,188 @@ class OmnipodSettingsViewController: UITableViewController {
+                    cell.textLabel?.text = LocalizedString("Finish pod setup", comment: "The title of the command to finish pod setup")
+                } else {
+                    cell.textLabel?.text = LocalizedString("Replace Pod", comment: "The title of the command to replace pod")
+                    cell.tintColor = .deleteColor
+                }
+
+                cell.isEnabled = true
+                return cell
+            }
+
+        case .diagnostics:
+            switch Diagnostics(rawValue: indexPath.row)! {
+            case .readPodStatus:
+                let cell = tableView.dequeueReusableCell(withIdentifier: NSStringFromClass(SettingsTableViewCell.self), for: indexPath)
+                cell.textLabel?.text = LocalizedString("Read Pod Status", comment: "The title of the command to read the pod status")
+                cell.accessoryType = .disclosureIndicator
+                return cell
+            case .playTestBeeps:
+                let cell = tableView.dequeueReusableCell(withIdentifier: NSStringFromClass(SettingsTableViewCell.self), for: indexPath)
+                cell.textLabel?.text = LocalizedString("Play Test Beeps", comment: "The title of the command to play test beeps")
+                cell.accessoryType = .disclosureIndicator
+                return cell
+            case .readPulseLog:
+                let cell = tableView.dequeueReusableCell(withIdentifier: NSStringFromClass(SettingsTableViewCell.self), for: indexPath)
+                cell.textLabel?.text = LocalizedString("Read Pulse Log", comment: "The title of the command to read the pulse log")
+                cell.accessoryType = .disclosureIndicator
+                return cell
+            case .testCommand:
+                let cell = tableView.dequeueReusableCell(withIdentifier: NSStringFromClass(SettingsTableViewCell.self), for: indexPath)
+                cell.textLabel?.text = LocalizedString("Test Command", comment: "The title of the command to run the test command")
+                cell.accessoryType = .disclosureIndicator
+                return cell
+            }
+
+        case .podDetails:
+            let podState = self.podState!
+            switch PodDetailsRow(rawValue: indexPath.row)! {
+            case .podAddress:
+                let cell = tableView.dequeueReusableCell(withIdentifier: NSStringFromClass(SettingsTableViewCell.self), for: indexPath)
+                cell.textLabel?.text = LocalizedString("Assigned Address", comment: "The title text for the address assigned to the pod")
+                cell.detailTextLabel?.text = String(format:"%04X", podState.address)
+                return cell
+            case .podLot:
+                let cell = tableView.dequeueReusableCell(withIdentifier: NSStringFromClass(SettingsTableViewCell.self), for: indexPath)
+                cell.textLabel?.text = LocalizedString("Lot", comment: "The title of the cell showing the pod lot id")
```

Comment:
I commented out a few descriptive properties and added TODOs to get this to compile.